exporter/http.go

```diff
@@ -12,6 +14,12 @@ import (
 )
 
 func (e *Exporter) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	err := e.verifyBasicAuth(r.BasicAuth())
```

Comment:
nit: you can combine this with line 18 for tighter scoping 
`if err := verifyBasicAuth(...); err != nil {`